content/articles/understanding-enum-types-in-java/index.md

```diff
@@ -0,0 +1,546 @@
+#[Java] Understanding Enum Types in Java: Building A Card Dealing and Shuffling App that  determines the ranking category of a player’s hand in a poker game
+
+Enums types are powerful tools in Java. Enums could be used to define a class of named constants and offer the advantage of type safety. They could also be used as keys in a switch statement or expression.
```

Comment:
Have an introduction to your article here. Simply explain what you are covering, its uses, etc